ncm-metaconfig/src/main/metaconfig/httpd/pan/schema.pan

```diff
@@ -7,11 +7,22 @@ include 'components/accounts/functions';
 type httpd_sslprotocol = string with match(SELF, '^\+?(TLSv1|TLSv1\.[012])$')
     || error("Use a modern cipher protocol, for Pete's sake!");
 
-type httpd_ciphersuite = string with match(SELF, '^(\+?TLSv1|!(RC4|LOW|[ae]NULL|MD5|EXP|3DES|IDEA))$')
+type httpd_ciphersuite = string with match(SELF, '^(\+?TLSv1|!(RC4|LOW|[ae]NULL|MD5|EXP|3DES|IDEA|SEED|CAMELLIA))$')
     || error("Use a modern cipher suite, for Pete's sake!");
 
-type httpd_nss_protocol = string with match(SELF, '^(TLSv1\.[012]|SSLv3|All)$');
-type httpd_nss_cipherstring = string with match(SELF, '^(\+|-)(rsa_3des_sha|rsa_des_56_sha|rsa_des_sha|rsa_null_md5|rsa_null_sha|rsa_rc2_40_md5|rsa_rc4_128_md5|rsa_rc4_128_sha|rsa_rc4_40_md5|rsa_rc4_56_sha|fortezza|fortezza_rc4_128_sha|fortezza_null|fips_des_sha|fips_3des_sha|rsa_aes_128_sha|rsa_aes_256_sha)$');
+# These are the settings for old clients, see https://access.redhat.com/articles/1467293 for stricter values.
+type httpd_nss_protocol = string with match(SELF, '^(TLSv1\.[012]|SSLv3)$');
```

Comment:
@JensTimmerman  please remove ';' as this is not the end of the line and won't compile